libs/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2019-2020 The STE||AR-Group
+# Copyright (c) 2020 The STE||AR-Group
```

Comment:
Not a big deal, but we can probably revert this change and have `2019-2020` in `create_library_skeleton.py`.